lib/std/debug.zig

```diff
@@ -1311,6 +1314,18 @@ pub const DebugInfo = struct {
 
         return obj_di;
     }
+
+    fn lookupModuleHaiku(self: *DebugInfo, address: usize) !*ModuleDebugInfo {
+        // TODO: implement me
+        var di = ModuleDebugInfo{
```

Comment:
This stub is useless and will crash right away if used.
You should be able to implement this in terms of `get_next_image_info` instead of waiting for someone to implement `dl_iterate_phdr`.

Does Haiku use ELF images and Dwarf debug info?

Reply:
i agree that the stub doesn't do much as is. Haiku does use the ELF format for binaries and does support dwarf debugging information (at least that is what gcc with debugging reported back)

thanks for the suggestion on not waiting around for `dl_iterate_phdr` support to roll around. implementing  with `get_next_image_info` is on my list of things i'd like to get to